development.md

```diff
@@ -0,0 +1,50 @@
+# Development
+
+### Example of Visual Code configuration for testing and debugging
```

Comment:
Maybe put this to `.vscode` folder? As I see, It's common practice, and it looks better then copy-pasting configs

Reply:
If we gonna have a common lanch config, then I think we need to set up common secrets(for aws, vault and etc). So that different devs could debug and test rattus